src/include/parser/statement_prepare.h

```diff
@@ -14,7 +14,7 @@
 
 #include "parser/sql_statement.h"
 #include "parser/statement_select.h"
-#include "expression/abstract_expression.h"
+#include "expression/parameter_value_expression.h"
 
 #include <algorithm>
 
```

Comment:
Yes this is used for prepared statments.
See https://www.postgresql.org/docs/9.3/static/sql-prepare.html

Although I don't know if it really works. I have never tried it.